tests/conftest.py

```diff
@@ -0,0 +1,50 @@
+"""Set up fixtures for unit tests."""
+
+import os
+import tempfile
+from logging import Logger
+from unittest.mock import Mock
+
+import h5py
+import pytest
+from varinfo import CFConfig, VarInfoFromNetCDF4
+
+
+@pytest.fixture
+def logger():
+    """Logger fixture."""
+    return Mock(spec=Logger)
+
+
+@pytest.fixture
+def mock_varinfo():
+    """VarInfo fixture mock."""
+    return Mock(spec=VarInfoFromNetCDF4)
+
+
+@pytest.fixture
+def smap_varinfo(sample_hdf5_file):
+    """VarInfo fixture for SMAP."""
+    return VarInfoFromNetCDF4(
+        sample_hdf5_file, config_file='hoss/hoss_config.json', short_name='SPL3FTA'
+    )
+
+
+@pytest.fixture(scope='function')
+def sample_hdf5_file():
+    """Create a temporary HDF5 file for testing."""
+    with tempfile.NamedTemporaryFile(suffix='.h5', delete=False) as tmp_file:
+        temp_filename = tmp_file.name
+
+    with h5py.File(temp_filename, 'w') as f:
```

Comment:
Cant you create a temp file with one of the libs already included? instead of adding a h5py dependency?  All the other tests look like they use Dataset()